cwms-data-api/src/main/java/cwms/cda/data/dto/CwmsDTO.java

```diff
@@ -14,7 +14,6 @@
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
 public abstract class CwmsDTO implements CwmsDTOBase {
     @Schema(description = "Owning office of object.")
-    @JsonProperty(required = true)
     protected final String officeId; // ALL DTOs require an office field
```

Comment:
Adding the required property is going to break other DTO's that are outside the scope of just the Outlet DTO's.  This has to do with the validator piece that was added, and the annotation reflection it does to check for required = true.  I think this needs to be a broader conversation on the design for this object, including name changes since it's not consistent.

I can create a ticket for that, but adding the annotation back in will be problematic right now.